code/__DEFINES/radio.dm

```diff
@@ -122,3 +124,8 @@
 #define REQ_DEP_TYPE_ASSISTANCE 	(1<<0)
 #define REQ_DEP_TYPE_SUPPLIES 		(1<<1)
 #define REQ_DEP_TYPE_INFORMATION 	(1<<2)
+
+//range of listening
+#define RADIO_HEARING_RANGE_BASIC_RADIO 3
+#define RADIO_HEARING_RANGE_INTERCOM 2
+#define RADIO_HEARING_RANGE_HEADSET 1
```

Comment:
```suggestion
#define RADIO_HEARING_RANGE_HEADSET 0
```
I feel as though just general hearing will begin to clutter up if you are, say, standing in a 2 by 2 formation with someone from sec, someone from engineering, etc. We should probably keep this the way it is for headsets.

Reply:
I really don't know but it's what Ruko requested. I will wait for opinions of how maintainers think.

Reply:
You shouldnt be able to hear someones headset if you are standing next to them, I think only if you are being carried by them or in the same locker as them/on the same tile. Implant radios should never be heard by others.

Reply:
oh wait this is the mic range

Reply:
I have no idea what Bacon is thinking. You triaged Ruko's issue.

Reply:
I don't think I can fix eavesdropping under someone's crotch, so do you want this to be 0 or what Ruko suggested?

Reply:
I think it should be 0 range rather than when standing next to them personally. Implants are not meant to be hearable by other people ever

Reply:
that's confusing. I didn't touch the hearing range of implants, but did to headsets. revert headsets'?

Reply:
Revert headsets to a 0 range I think